source/guides/1.0/getting-started.md

```diff
@@ -256,7 +256,7 @@ Finished in 0.011854s, 84.3600 runs/s, 168.7200 assertions/s.
 Let's use our new knowledge about the major Hanami components to add a new action.
 The purpose of our Bookshelf project is to manage books.
 
-We'll store books in our database and let the user manage them with our project.
+We'll store books in our database and let the user manage them with our application.
```

Comment:
in hanami we use project for full repository and application for part of the project in `apps/` folder. In this case, `project` better raise than `application`

Reply:
ah i understand. but here 'application' as in software application that enables us to do something. so example Word is an application that allows us to write documents or Bookshelf in this case is an application that helps us store books. 'application' here doesn't mean '/apps' folder, it means the [program/software/web app](https://en.wikipedia.org/wiki/Application_software) we're building. but i can change it back if you're still not convinced.  @davydovanton